packages/slice-machine/src/legacy/components/Forms/CreateCustomTypeModal/CreateCustomTypeModal.tsx

```diff
@@ -165,10 +165,18 @@ export const CreateCustomTypeModal: React.FC<CreateCustomTypeModalProps> = ({
           })} name is already taken.`;
         }
 
+        if (["update", "insert"].includes(label.toLowerCase())) {
```

Comment:
💡 This could also be extracted, like you did for the slice name

```suggestion
        if (RESERVED_SLICE_NAME.includes(label.toLowerCase())) {
```